### ISSUE TITLE
 Standard names: renaming of atmosphere_relative_vorticity to atmosphere_upward_relative_vorticity

### ISSUE THREAD

**Issue by GeyerB** [cf-convention/discuss#38]
**Proposer's name** Beate Geyer / Ronny Petrik
**Date** 2020/03/11

A revision of all vorticity entries seems to be needed, starting here:

**- Term** atmosphere_upward_relative_vorticity
**- New Definition** The vorticity is a vector quantity. "Upward" indicates a vector component which is positive when directed upward (negative downward). Atmosphere upward relative vorticity is the vertical component of the 3D air vorticity vector. The vertical component arises from horizontal velocity only. “Relative” means only the vorticity of air and does not consider the Earth’s rotation. In contrast, the “atmosphere_upward_absolute_vorticity” considers the Earth’s rotation.
**- Units** s-1

**Comment by feggleton** [comment]
Hi Beate and Ronny,

Thanks for your proposal. Just for the future, if the changes you are proposing are for a set of names with similar changes (ie. add upward to vorticity names and amend definitions) we are happy for these to be in one issue. The changes here would be:

From atmosphere_relative_vorticity to atmosphere_upward_relative_vorticity
Description from: Relative vorticity is the upward component of the vorticity vector i.e. the component which arises from horizontal velocity.

to: The vorticity is a vector quantity. "Upward" indicates a vector component which is positive when directed upward (negative downward). Atmosphere upward relative vorticity is the vertical component of the 3D air vorticity vector. The vertical component arises from horizontal velocity only. “Relative” means only the vorticity of air and does not consider the Earth’s rotation. In contrast, the “atmosphere_upward_absolute_vorticity” considers the Earth’s rotation.

It would be great to get some discussion going here about the changes to the vorticity names (there are 6 in the standard name table), if the definitions are appropriate and if people think this detail is needed for clarification. Ensuring that the changes do not change the meaning of the names (aliases can be created here as long as we are not changing the meaning, otherwise this would be a new name). This term is now in the cfeditor.

Thanks

**Comment by feggleton** [comment]
To match with #40 we will re-arrange the definition of atmosphere_upward_relative_vorticity to be (if that works here):

Atmosphere upward relative vorticity is the vertical component of the 3D air vorticity vector. The vertical component arises from horizontal velocity only. “Relative” means only the vorticity of air and does not consider the Earth’s rotation. In contrast, the “atmosphere_upward_absolute_vorticity” considers the Earth’s rotation. The vorticity is a vector quantity. "Upward" indicates a vector component which is positive when directed upward (negative downward).

These terms will need to be accepted together as they mention the other. Looking at the absolute term discussion in #40 this seems to be an acceptable and simple addition. Are there any comments on this change?

**Comment by JonathanGregory** [comment]
Dear Francesca

I think it's good to be clear about the sign convention for vorticity. It is correct to to describe it as a vector with a vertical coordinate that is positive upward, as you have, but it would probably be helpful also to describe it as being positive for anticlockwise rotation when viewed from above.

Best wishes

Jonathan

**Comment by japamment** [comment]
Dear Beate, Ronny, Francesca, Jonathan, @GeyerB @feggleton @JonathanGregory 

It looks like agreement has been reached on this proposal to create the following alias:
**atmosphere_relative_vorticity** -> **atmosphere_upward_relative_vorticity** (Canonical unit: s-1).
The definition has been clarified and will read as follows:
'Atmosphere upward relative vorticity is the vertical component of the 3D air vorticity vector. The vertical component arises from horizontal velocity only. "Relative" means only the vorticity of the air is considered, excluding the Earth's rotation. In contrast, the quantity with standard name atmosphere_upward_absolute_vorticity includes the Earth's rotation. "Upward" indicates a vector component which is positive when directed upward (negative downward). A positive value of atmosphere_upward_relative_vorticity indicates anticlockwise rotation when viewed from above.'

This change is accepted for publication in the standard name table and will be added in the September update. I agree it makes sense to accept [#40](https://github.com/cf-convention/discuss/issues/40) (atmosphere_upward_absolute vorticity) at the same time as this one.

Best wishes,
Alison

**Comment by japamment** [comment]
Dear Beate, Ronny, Francesca, Jonathan, @GeyerB @feggleton @JonathanGregory

I have just looked at issue [#39](https://github.com/cf-convention/discuss/issues/39) (x and y relative_vorticity) and seen the comment from @taylor13 regarding the definition of "relative". I think Karl's wording is better, so to make all the names consistent I have changed this one slightly to:
'Atmosphere upward relative vorticity is the vertical component of the 3D air vorticity vector. The vertical component arises from horizontal velocity only. "Relative" in this context means the vorticity of the air relative to the rotating solid earth reference frame, i.e. excluding the Earth's own rotation. In contrast, the quantity with standard name atmosphere_upward_absolute_vorticity includes the Earth's rotation. Vorticity is a vector quantity. "Upward" indicates a vector component which is positive when directed upward (negative downward). A positive value of atmosphere_upward_relative_vorticity indicates anticlockwise rotation when viewed from above.'

(The name will still be included in today's update).

Best wishes,
Alison
